### DISCUSSION TITLE
How do you solve the tooltip disappearing when pressing outside one of the bars?

### DISCUSSION THREAD

**Discussion by Vasault** [Abhinandan-Kushwaha/react-native-gifted-charts#912] [Q&A]
When you enable the tooltip and you press on a bar, it highlights the bar and shows the tooltip above it, but when you press somewhere else, the tooltip disappears and the bar is still highlighted

**Comment by Abhinandan-Kushwaha** [comment]
Hi @Vasault I am unable to repro this issue. If I press somewhere else, the tooltip disappears and the bar is also unhighlighted. See this snack- https://snack.expo.dev/@ak_97/bar-highlight-tooltip
